processes/issue-triage.md

```diff
@@ -0,0 +1,95 @@
+# Issue Triage
+
+## What is triaging?
+Triaging is a process where maintainers review, respond to, and organize GitHub issues and pull requests. Triaging involves categorizing issues and pull requests based on factors such as kind (bug, enhancement, etc.) and priority/urgency. At the end of this process, new issues will be categorized appropriately and team leaders will be able to make backlog prioritization decisions, as needed.
+
+## Why is triaging important?
+Triaging is important because it:
+- promotes a responsive and inclusive community
+- leads to greater transparency, better discussions, and more collaborative, informed decision-making
+- speeds up issue management
+- prevents work from lingering endlessly
+- helps build prioritization, negotiation, and decision-making skills, which are critical to most tech roles.
+
+## Who is responsible?
+The responsibility is open to anyone. If no one volunteers to triage issues, it falls on the maintainers of each tool/repo. Maintainers are listed in the `MAINTAINERS.md` file of each repo.
+
+## When do we triage?
+Triage can happen asynchronously and continuously, or at regularly scheduled times. It's left up to the maintainers of each repo to figure out what's best for them.
+
+Some good practices:
+- enable GitHub notifications so that you're notified of activity
+  - filter these notifications appropriately so that you're not overwhelmed
+- schedule regular blocks of time on your calendar to triage (for example, block 30 minutes every morning or 1-2 hours on Tuesdays and Thursdays)
+- consider assigning a triager for a day or week at a time
+
+## How do we triage?
+1. [Respond to Newly Created Issues and PRs](#1-respond-to-newly-created-prs-and-issues)
+2. [Triage Issues by Type](#2-triage-issues-by-type)
+3. [Define Priority](#3-define-priority)
+
+### 1. Respond to Newly Created PRs and Issues
+Labels are the primary tools for triaging. New issues are automatically assigned a `carvel triage` label. Issues with the `carvel triage` label indicate that the issue is awaiting triage.
+
+1. Filter the _PRs_ with a `carvel triage` label for the repos you're focused on. If you're using the ZenHub board, these will show up in the Needs Review column.
```

Comment:
Currently we don't have a way to label new PRs with `carvel-triage`, they just appear in the zenhub 'needs review' column. We could 
1. Build automation to label new PRs
2. Manually label new PRs with `carvel-triage` when we notice them
3. Update this line to mention that new PRs appear in the 'needs review' column

I suggest (3) for the time being, until we run into issues with the current process

Reply:
@cari-lynn Ah, I didn't realize that. That's good to know. I agree with your suggestion of the third option for now.